lib/parser.rb

```diff
@@ -1,12 +1,26 @@
+require 'json'
+
 class Parser
   def initialize(file)
     @file = file
-    unless File.exist?(@file)
+    unless File.exist?(file)
       raise StandardError.new "Arquivo não encontrado"
+    else
+      @data = File.readlines(file)
     end
   end
     
   def get_first_line
-      data = File.open(@file, &:readline)
+    first = @data.first
+  end
+
+  def line_count
+    count = @data.size
+    result = {
+      "games.log": {
+        "lines": count      
```

Comment:
```suggestion
        "lines": @data.size      
```

Da pra vc simplificar e chamar direto aqui sem precisar atribuir a uma váriavel